app/templates/components/forms/events/view/create-access-code.hbs

```diff
@@ -11,28 +11,8 @@
     <Input
       @type="number"
       @name="number_of_access_tickets"
-      placeholder="0"
       @min={{0}}
```

Comment:
change to ```"0"```

Reply:
And also mention which Issues u r solving in description

Reply:
Should I remove placeholder???

Reply:
yes

Reply:
Also Pls refer the #5750

Reply:
Thank U I have fixed it